src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,53 @@
+package coffee.order;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class CoffeeOrderBoard {
+
+    private List<Order> orders;
+    private Map<Integer, Order> orderMap;
```

Comment:
generating order numbers based on Map will lead to different orders will have have number, that is not good, Try to implement it with the queue